etl/kpinetwork_analizer.py

```diff
@@ -82,6 +82,147 @@
 )
 
 
+def read_ids_from_database(env, db_table):
+    database = "kpinetworkdb"
+    if env == "demo":
+        database = "demokpinetworkdb"
+
+    ctg_connection = "{}_connection".format(env)
+    catalog_connection = glueContext.extract_jdbc_conf(ctg_connection)
+
+    dynamic_frame = glueContext.create_dynamic_frame.from_options(
+        connection_type="postgresql",
+        connection_options={
+            "url": "{}/{}".format(catalog_connection["url"], database),
+            "user": catalog_connection["user"],
+            "password": catalog_connection["password"],
+            "dbtable": db_table,
+        },
+    )
+    dynamic_frame = dynamic_frame.toDF().createOrReplaceTempView(
+        "{}_table".format(db_table)
+    )
+    sqldf = spark.sql("SELECT id FROM {}_table".format(db_table))
+    result_from_sqldf = sqldf.collect()
+    ids_from_db = [row.id for row in result_from_sqldf]
+    return ids_from_db
+
+
+def exists_metric_from_file(env, company_id, scenario_name, metric_name):
+    resultant_metric_id = None
+    database = "kpinetworkdb"
+    if env == "demo":
+        database = "demokpinetworkdb"
+
+    ctg_connection = "{}_connection".format(env)
+    catalog_connection = glueContext.extract_jdbc_conf(ctg_connection)
+
+    dynamic_fs_frame = glueContext.create_dynamic_frame.from_options(
+        connection_type="postgresql",
+        connection_options={
+            "url": "{}/{}".format(catalog_connection["url"], database),
+            "user": catalog_connection["user"],
+            "password": catalog_connection["password"],
+            "dbtable": "financial_scenario",
+        },
+    )
+
+    dynamic_fs_frame = dynamic_fs_frame.toDF().createOrReplaceTempView(
+        "financial_scenario_table"
+    )
+    fs_dataframe = spark.sql(
```

Comment:
It would be better a name like dynamic_scenario_frame, scenario_dataframe

Reply:
also changed.